GabrielaRamirez/EjerciciosHackerrank/src/hackkerank/day1/StdinII.java

```diff
@@ -0,0 +1,15 @@
+package hackkerank.day1;
+import java.util.Scanner;
+
+public class StdinII {
+    public static void main(String[] args) {
+        Scanner scan = new Scanner(System.in);
+        int a = scan.nextInt();
```

Comment:
Remember, variable names and parameter names must be meaningful:
https://google.github.io/styleguide/javaguide.html#s5.2.7-local-variable-names